e2e/pages/ordersNeeded/ordersNeeded.js

```diff
@@ -0,0 +1,30 @@
+const I = actor();
+
+module.exports = {
+  fields: {
+    ordersAndDirections: {
+      careOrder: 'Care order',
+      supervisionOrder:'Supervision order',
+      educationSupervisionOrder: 'Education supervision order',
+      emergencyProtectionOrder: 'Emergency protection order',
+      other: 'Other',
+    },
+    directionAndInterim: '#orders_directionsAndInterim',
+  },
+
+  fillTextArea(testString = 'Test string') {
```

Comment:
just a question @Jamesnelson117. what are your thoughts on making this a bit more descriptive? Sure you're filling a text area but a new dev coming to this. Do I understand what text area? Open to discussion 😄

Reply:
I’d have to agree with you, ‘enterDirectionAndInterim’ is more explicit, would you agree? Happy to change if so.  

I’d a look through the codebase and there’s a few instances were the UI element has influenced the test name. I.E selectTab(), hallFillForm() could equally add some confusion to a new dev. Perhaps one to include in the upcoming tech debt story?